### PR TITLE
feature(update) update to AppVersion 0.26.1

### LINKED ISSUE #67: prepare helm-chart for upstream flux2 0.26 release
prepare helm-chart for upstream flux2 0.26 release
https://github.com/fluxcd/flux2/issues/2308

### PR DESCRIPTION
**Pull request by dwerder**
Signed-off-by: Daniel Werdermann <daniel.werdermann@gmail.com>

#### What this PR does / why we need it:

- Update to upstream 0.26.1
- Mostly update securityContext in teh deployments

#### Which issue this PR fixes
  - fixes #67 

#### Checklist

- [x] [DCO](https://github.com/fluxcd-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] helm-docs are updated
- [x] Helm chart is tested
- [x] Update artifacthub.io/changes in Chart.yaml
- [x] Run `make reviewable`

### CODE REVIEW

**Comment by stefanprodan** [comment]
It’s time to deprecate the multi-tenancy chart and offer here an option for platform admins to lock down Flux. Please see which flags are needed to accomplish this: https://fluxcd.io/docs/installation/#multi-tenancy-lockdown

File: charts/flux2/values.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 installCRDs: true
 
+# -- Implement the patches for Multi-tenancy lockdown. 
+# See https://fluxcd.io/docs/installation/#multi-tenancy-lockdown
+multiTenancyLockdown: false
```
Comment (stefanprodan) [inline charts/flux2/values.yaml:7]: Can we please allow users to change the default service account. Something like this:

```yaml
multitenancy:
  enabled: false
  defaultServiceAccount: "default"
```

Comment (dwerder) [reply charts/flux2/values.yaml]: Done

File: charts/flux2/values.yaml [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 installCRDs: true
 
+multiTenancyLockdown:
```
Comment (stefanprodan) [inline charts/flux2/values.yaml:5]: Let's rename this to `multitenancy`

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dwerder 

I would change the PR title to "Update Flux to v0.26.1"

**Comment by dwerder** [comment]
Already done. The PR uses 0.26.1 now.

**Pull request merged**